### DISCUSSION TITLE
Sablier as a reverse proxy

### DISCUSSION THREAD

**Discussion by webysther** [sablierapp/sablier#168] [Ideas]
I tried using the project but I get stucked when need to install the module, my reverse proxy ([NPM](https://github.com/NginxProxyManager/nginx-proxy-manager)) are not so friendly to changes as needed, so I move to a alternative called [container nursery](https://github.com/ItsEcholot/ContainerNursery) which is insane more friendly but is not maintained nor get fixed a [annoyng bugs](https://github.com/ItsEcholot/ContainerNursery/issues/48).

The idea is became a simple reverse proxy to the real app (which is already behind a proxy) and check the container based on sock, when user access the url of project the _modus operandi_ is the same but more pratical.

Will the nice this project have a "self contained provider" like this project, this will make the plugin not necessary anymore.

Advantages of this method:

- No need to change the real reverse proxy
- Change only the proxy pass to point to the fake reverse proxy

**Comment by acouvreur** [comment]
I could definitely bake Sablier with a reverse proxy built in, however it would be something very naive, very simplistic. Things such as auto SSL would not be part of this.

In the end I think it's to hard to build and maintain a reverse proxy in the same project. What do you think?

**Comment by webysther** [comment]
For sure is simplistic, this is why I open another discussion to go deeper about this question, I will go to another one.
